authorization-common/src/main/java/com/sequenceiq/authorization/service/ResourcePermissionChecker.java

```diff
@@ -15,6 +16,9 @@
 
     protected Map<String, AuthorizationResourceAction> getAuthorizationActions(String resourceCrn, AuthorizationResourceAction action) {
         ResourceBasedCrnProvider resourceBasedCrnProvider = commonPermissionCheckingUtils.getResourceBasedCrnProvider(action);
+        if (resourceBasedCrnProvider == null) {
+            return Collections.emptyMap();
+        }
```

Comment:
could you add an else branch or restructure the code so you won't have a return in the middle of the method?

Reply:
I think it is much clearer then if I create an if {} else {} structure. Otherwise, we have a lot of similar code style in the codebase.